### DISCUSSION TITLE
Search Results showing results on page load

### DISCUSSION THREAD

**Discussion by Pbear74** [microsoft-search/pnp-modern-search#3897] [Q&A]
I have two search results web parts on my search page, one for regular SharePoint data and another for people. On page load both web parts will load with data showing, how do I prevent that from happening? I want the page to load with nothing but the search bar.

**Comment by kasperbolarsen** [comment]
If you have the {searcTerms} in your Query and haven't set a default value for the search box on page 3 in the results web part, you should not get any results ( based on the assumption that you are running version 4.12.2 )

**Reply by Pbear74** [reply]
Yes, that is the version I am running.  Page 1 has "{searchTerms}" and there is no default value in page 3.

**Reply by kasperbolarsen** [reply]
And there is nothing but {searchTerms} ?

**Reply by Pbear74** [reply]
correct

**Comment by kasperbolarsen** [comment]
And you using LocalPeopleResult and localSharePointResult in the datasource choice?

**Reply by Pbear74** [reply]
I am using local SharePoint results as the data source.  I just disabled audience targeting and that cleared the results on load. Why would that cause the results to load on page load?

**Comment by kasperbolarsen** [comment]
Strange, what does the Dev tool say about it? ( see this short yt video if you are not familiar with using the network tab to inspect traffic, https://www.youtube.com/watch?v=kPMSiRxK8H8 )

**Reply by kasperbolarsen** [reply]
so when the Audience targeting is enabled the modified:
![image](https://github.com/user-attachments/assets/bf0539ce-4389-4beb-bc05-be1fa7bc5d4e)

not sure if this is a bug in our code or just "by design"
